__test__/update-password.test.js

```diff
@@ -0,0 +1,92 @@
+const request = require('supertest');
+import app, { connectDB } from '../src/app';
+const User = require('../src/models/user');
+
+connectDB(); // connect database
+
+describe('Update password feature', () => {
+  let user;
+  let token;
```

Comment:
you can use only one line   => `let user, token`

Reply:
> you can use only one line => `let user, token`

Fixed